docs/en/modules/admin/pages/participants/authorizations.adoc

```diff
@@ -1,32 +1,20 @@
 = Authorizations
 
-This section allows you to manage participants' authorizations. It depends on which Authorizations you have configured and
-enabled by your developers. To understand more what are authorizations and how to configure them, please refer to the
-xref:customize:authorizations.adoc[Authorizations section in the Customize Guide].
-
-With authorizations an administrator can define different permissions so participants can make some actions in a component.
-For instance you can configure that only participants who have verified herselves by their identity documents can vote on
-proposals in a participatory process.
+Authorizations allow to give specific participation rights to participants who have completed a verification process. 
+This verification process can be direct, have multiple steps, be online or in-person. 
 
-A participant can see which authorizations do they have available in their account settings.
+For instance you can configure that only participants who have verified themselves by their identity documents can vote on
+proposals in a specific participatory process.
 
-image:authorizations_account.png[Authorizations in participants account]
-
-== Authorizations revocation
+NOTE: Several authorizations are available by default, but you can also develop your own authorizations. 
 
-A default Decidim installation will not have verified participants.
+== Enable / disable authorizations
 
-image:authorizations_admin_default.png[Authorizations in a default installation]
-
-In the moment that there are some participants verified, an administrator can revoke all the authorizations, as the attributes
-that give permissions to those participants could be changed in time and the platform didn't receive these changes in the
-participants statuses. For instance, an identity document of a participant could be expired, or if a participant have been
-authorized by the "code by postal letter", that participant could move in the meantime.
-
-image:authorizations_admin_revocation.png[Authorizations revocation]
+To enable an authorization, you need to have access to the xref:configure:system.adoc[System panel] or ask your administrator. 
```

Comment:
Can we find another naming for "instance administrator", as it is written could be confusing with "administrators".
Maybe "implementer" is better, and I see we are using it below.

Reply:
Totally, I updated it and will call it that way in all the other articles!